src/curated-corpus/components/SidebarDatePicker/SidebarDatePicker.tsx

```diff
@@ -30,22 +28,27 @@ export const SidebarDatePicker: React.FC<SidebarDatePickerProps> = (
 ): JSX.Element => {
   const { handleDateChange, selectedDate } = props;
 
+  //TODO @Herraj remove datepicker icon and enable datepicker drop down on text field click
   return (
```

Comment:
We've agreed on Slack that it's not necessary - what is needed here is communicate that change to the curators when we're ready to show them the updated app.